### DISCUSSION TITLE
How to mapping object in 2D image to point cloud group?

### DISCUSSION THREAD

**Discussion by ThanhTrung128** [UTokyo-FieldPhenomics-Lab/EasyIDP#84] [Q&A]
I would like to mapping object in 2D image to point cloud group 
For example, in case of a small box is appeared in 2D image, which is source file of easyIDP.
I would like to extract dense point group corresponding to that small box from all point cloud

Is it possible in EasyIDP?
Thank you for your cooperation.

**Comment by HowcanoeWang** [comment]
The 2D image is the Orthomosaic (DOM) or the UAV raw images?

If is the bbox/polygon on DOM, the EasyIDP fully supports cropping out the corresponding regions from the dense point cloud. Please kindly check this documentation: https://easyidp.readthedocs.io/en/latest/jupyter/crop_outputs.html#Read-and-crop-point-cloud

If is the deep learning detection/segmentation results (bbox/polygon) on UAV images, the EasyIDP currently has not provided a neat API for doing this easily (but provide a complex and not 100% accurate solution here: https://github.com/UTokyo-FieldPhenomics-Lab/EasyIDP/discussions/44, which convert the bbox/polygon from UAV image to DOM). If possible, please try detection/segmentation directly on DOM, unless the DOM quality is too bad.
